test/utils/ValidatorTester.js

```diff
@@ -1,8 +1,19 @@
 const HTTPStatus = require("http-status-codes");
+
 const { ErrorTypes } = require("../../src/api/middleware/errorHandler");
 const ValidationReasons = require("../../src/api/middleware/validators/validationReasons");
 const { DAY_TO_MS } = require("../utils/TimeConstants");
 
+/**
+ * Checks the common parts of the error response: the status code, error_code property and that the body also has the error properties
+ * @param {*} res The response object
+ */
+const checkCommonErrorResponse = (res) => {
```

Comment:
Shouldn't this be `checkValidationErrorResponse`? The first thought I had when reading was that this would check for 500 errors, as they are the common/generic. Since this checks for 422 (specific request validation error), the function name should be clearer

Reply:
As the doc comment mentions, this "Checks the common parts of the error response". This method was created to reduce the code duplication since there were some parts that were repeated along this file.

Since this does not check the full response, and only the parts that are common in the response between all of the validator tests, this name was chosen. As such, I see no issue with it, and think that `checkValidationErrorResponse` would be misleading as it would seem like it would check for all of the response, but only performs checks on a small part of it, since the separate tests below perform the remaining checks.

Reply:
Furthermore, since this function is only used in this file, has a doc comment that explains it, and is not even exported from the file, I think this is pointless bikeshedding :sweat_smile:

Reply:
I guess it could be renamed to `checkCommonValidationErrorResponse` but that is just a mouthful, and sincerely irrelevant given the reasons above.

Reply:
Sure, since this is a single-file thing (not exported), used to reduce repetition and the file is the validation one, the renaming is not as important, let's ship this